sw/device/lib/dif/dif_aes.h

```diff
@@ -147,12 +196,12 @@ typedef enum dif_aes_manual_operation {
    * AES operates in automatic mode - which means that the encryption/decryption
    * is automatically triggered on every successful `dif_aes_*_load_data()`.
    */
-  kDifAesManualOperationAuto = 0,
+  kDifAesManualOperationAuto = 1,
   /**
    * AES operates in manual mode - which means that the encryption/decryption
    * is manually triggered by `dif_aes_trigger(kDifAesTriggerStart)`.
    */
-  kDifAesManualOperationManual,
+  kDifAesManualOperationManual = (1 << 1),
```

Comment:
This should just be `1` instead of `0x2`.

Reply:
Done